.eslintrc

```diff
@@ -1,3 +1,6 @@
 {
-  "extends": "@ahmdigital/eslint-config/react-config"
+  "extends": "@ahmdigital/eslint-config/react-config",
+  "rules": {
+    "jest/no-deprecated-functions": "off"
+  }
```

Comment:
This rule override is here to prevent a lint error that occurs because `jest` is not installed in this library